include/libtorrent/disk_buffer_pool.hpp

```diff
@@ -96,7 +96,7 @@ namespace libtorrent {
 		}
 		int num_to_evict(int num_needed = 0);
 
-		void set_settings(aux::session_settings const& sett);
```

Comment:
I see. this isn't really a polymorphic type in that sense. but I see what you mean. Both functions are called ``set_settings``.